intro-to-tests/problem-set-intro-to-pytest.md

```diff
@@ -277,6 +323,20 @@ In tests, long test names are welcome. For each test case, the best test names u
 
 The following guideline is challenging to master, but the best test names tend to describe a situation, and not name exact variable names. If the variable name changes, then the test name could become inaccurate.
 
+Pytest will only recognize test names beginning with `test_` or ending with `_test`.
+
+##### !end-hint
+
+##### !explanation
+
+The test name `test_calculate_dividing_by_zero_returns_error_msg()` is the best option for this test because it describes the condition for which we want to test the most accurately.
+
+The option `test_calculate_dividing_when_second_number_is_zero()` is not the best option here because it does not clearly indicate what the function is intending to return in the case of dividing by zero.
+
+The option `test_calculate_dividing_num_b_is_zero()` is not the best option here because it does not clearly indicate what the function is expected to return in the case of `b` being zero.
+
+The option `test_calculate_dividing_by_zero()` is not the best option here because, again, it does not clearly indicate what the function is expected to return if dividing by zero.
+
 ##### !end-explanation
 ### !end-challenge
 <!-- prettier-ignore-end -->
```

Comment:
Further down in this file (around line 949) there's a misspelling of `robust` as `robost`. As long as you're here, could you update that as well?

Reply:
Fixed 👍🏾 Thank you for taking a look!!